crawl-ref/source/tilepick.cc

```diff
@@ -2573,12 +2606,16 @@ tileidx_t tileidx_item(const item_def &item)
             return _tileidx_corpse(item);
 
     case OBJ_ORBS:
+        if (item.quantity <= 0)
```

Comment:
I guess the crawl idiom here and below would be to use a single return statement with `? ... : ...`. (I'm just going to merge this PR though, this comment is only for reference.)

Reply:
Yup; I think I didn't do that because of the resulting line length (then the one below was for consistency).